.github/workflows/ci-repo.yml

```
@@ -10,24 +10,36 @@ on:
     branches:
       - "main"
       - "release-*"
-
   workflow_dispatch:
 
   workflow_call:
```

Please update these so we can use them to dictate when we publish.
```suggestion
  workflow_dispatch:
    inputs:
      shouldRelease:
        description: 'Should a release be created?'
        required: false
        default: 'false'

  workflow_call:
    inputs:
      shouldRelease:
        description: 'Should a release be created?'
        required: false
        default: 'false'
```